runtime/tests/port/main.c

```diff
@@ -296,114 +296,110 @@ signalProtectedMain(struct J9PortLibrary *portLibrary, void *arg)
 	}
 
 	rc = 0;
-	if (serverName[0]) {
```

Comment:
I think the local variable should also be removed along with the option that sets it.

Reply:
Updated.